pygmt/clib/session.py

```diff
@@ -1795,7 +1795,8 @@ def virtualfile_to_dataset(
         ...             ) as vouttbl:
         ...                 lib.call_module("read", f"{tmpfile.name} {vouttbl} -Td")
         ...                 result = lib.virtualfile_to_dataset(
-        ...                     output_type="file", vfname=vouttbl
+        ...                     vfname=vouttbl,
+        ...                     output_type="file",
```

Comment:
Should be ok to put this in a single line right?
```suggestion
        ...                     vfname=vouttbl, output_type="file"
```